src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -221,10 +207,12 @@ export class ViolationTicketService {
         ticketNumber: this.ticket.ticket_number,
         time: this.datePipe.transform(this.ticket.issued_date, "HH:mm"),
       }
-      if(this.dateDiff(this.ticket.issued_date)<30){
+      if (this.dateDiff(this.ticket.issued_date) < 30) {
```

Comment:
I wonder if this should be ' <= 30'? You have up to 30 days from the date of service to submit a dispute. So if you are on the 30th day you should still be able to submit? If I was issued a ticket on Jan 1, I could still dispute it up to the end of Jan 31 cause there are 30 days between Jan 1 and Jan 31.  Day 1 is at the end of the day after you were issued the ticket.